docs/UserGuide.adoc

```diff
@@ -19,16 +19,63 @@ By: `CS2103-AY1819S2-W10-1`      Since: `Jan 2019`      Licence: `MIT`
 == Introduction
```

Comment:
We should do a slight refactor of this based on the feedback we got from Liang Jun's team.